content/creator/worlds/about.md

```diff
@@ -90,3 +90,23 @@ A couple of optional custom settings can be specified in the `scene.json` when d
   }
 }
 ```
+
+### Access Control Lists (ACL)
+
+When a team (more than one person) are contributing to the development of a scene, it may be beneficial to have each contributor have the ability to publish the scene under a single NAME. As stated above, the NAME owner is the only one allowed to run such deployment.
+
+So we have introduced the concept of Access Control List (or ACL for short). The idea is that the owner of the NAME can grant other wallets permission to publish a scene under his NAME. This way the whole team (or a group of selected members) can be added to the world ACL and those will be able to publish the scene.
```

Comment:
I would put this in passive voice: The concept of Access Control List was introduced.